uwblueprint/project-read-frontend
# add enrolment status columns

### Notion ticket link



[Client enrolment status columns](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=29330407688445eb904d18b8d4c670bb)



### Implementation description

- Added 3 columns: enrolled, current_class and status (always appearing as last 3 columns)



### Steps to test

1. Run backend (* might need to run this guy if it's not merged into main yet: https://github.com/uwblueprint/project-read-backend/pull/64)
2. 👁  ensure it looks okay in the frontend



### What should reviewers focus on?

- Code quality

### Checklist
